src/pages/colegio.astro

```diff
@@ -1,14 +1,51 @@
 ---
-export function getStaticPaths() {
-  return [];
-}
+import Rocket from "../assets/3DRocket.png";
+import Divider from "../components/atoms/Divider.astro";
+import CountDown from "../components/organisms/CountDown.astro";
+import FAQ from "../components/faq/FAQ.astro";
+import CTAFull from "../components/sections/CTAFull.astro";
+import CondritaSection from "../components/sections/CondritaSection.astro";
+import NeedKnowSection from "../components/sections/NeedKnowSection.astro";
+import PrizesSections from "../components/sections/PrizesSections.astro";
+import ReasonsSection from "../components/sections/ReasonsSection.astro";
+import SchoolFormSection from "../components/sections/SchoolFormSection.astro";
+import SponsorsSection from "../components/sections/SponsorsSection.astro";
+import TimelineSections from "../components/sections/TimelineSections.astro";
+import WhySection from "../components/sections/WhySection.astro";
+import Layout from "../layouts/Layout.astro";
 ---
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta http-equiv="refresh" content="0;url=/es/colegio/" />
-</head>
-<body>
-  <p>If you are not redirected, <a href="/es/colegio/">click here</a>.</p>
-</body>
-</html>
+
+<Layout title="Condrita Space Apps">
+	<main class="w-full h-full scroll-smooth">
+		<CTAFull 
+			fullHeight
+			background={{from: "#105bd8", to: "#0b3d91"}}
+			href="#school-form"
+			title="Contrita Campamento de Verano Espacial "
+			body={`¡La aventura te espera!\nInscríbete al Campamento Espacial más grande del país.`}
+			image={Rocket}
+		/>
+		<!-- <StoryblokComponent blok={content} /> -->
+		<Divider />
+		<CountDown
+			countdownDate="jul 11, 2024 10:00:00"
+		/>
```

Comment:
Is this date modifiable in the cms? and there is a data validator or error handling?

Reply:
Yes, right now we only want to test the cms with test-post route. And the cms is responsible for that validation